### ISSUE TITLE
Leaflet 1.0 support?

### ISSUE THREAD

**Issue by speller** [SpatialServer/Leaflet.MapboxVectorTile#78]
Tried this plugin with the leaflet 1.0.3 and got error that `L.TileLayer.Canvas` is undefined.

**Comment by leafyeh7** [comment]
Leaflet.VectorGrid supports Leaflet 1.x

https://github.com/Leaflet/Leaflet.VectorGrid

**Comment by michaelborn** [comment]
Yep, also getting this error. It appears that `L.TileLayer.Canvas` was removed from Leaflet in 2015.
https://github.com/mapbox/leaflet-image/issues/42
